### PR TITLE
Make the meetup iframe responsive friendly

### PR DESCRIPTION
**Pull request by danybmx**
This makes that the meetup iframe stay inside the sidebar and keeps centered on it.

### CODE REVIEW

**Comment by antonmry** [comment]
Wow! This is awesome! Thanks @danybmx !!

I will assign to @tomasalmeida who was working on the styles and design web!

**Comment by tomasalmeida** [comment]
Thanks @danybmx, just sent one question by slack to you. I seems the PR perfect, just want to confirm one thing before merging it :)

**Comment by tomasalmeida** [comment]
Just have a talk with @danybmx and we decided to keep this PR on hold until new commits.

**Review by tomasalmeida — APPROVED** [review]

**Pull request merged**
